### ISSUE TITLE
Docket number out of sync between Elastic and DB

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#3879]
The search here:

https://www.courtlistener.com/?type=r&q=&type=r&order_by=score%20desc&docket_number=4%3A22-cv-05246

Looks for docket number 4:22-cv-05246, and finds a result:

![image](https://github.com/freelawproject/courtlistener/assets/236970/028e0c85-6fc0-48ac-b148-687ab376af44)

 When you click the result, it takes you to a docket where the number is different: 3:22-cv-05246:

https://www.courtlistener.com/docket/65349966/golden-v-google-llc/

![image](https://github.com/freelawproject/courtlistener/assets/236970/8cde91bd-21fa-455c-ab4e-aab2e05d9e78)

I thought we had fixed this in Elastic, so I'm surprised to have a user notice this. Is there any way we can diagnose this? I think what happened is that the docket number changed, but why didn't we get the update properly?

**Comment by albertisfu** [comment]
I investigated this issue and I found that the docket number changed between:

[Nov. 28, 2023, 1:27 p.m](https://www.courtlistener.com/admin/recap/pacerhtmlfiles/7407279/change/) (PT) and  
[Nov. 28, 2023, 2:38 p.m](https://www.courtlistener.com/admin/recap/pacerhtmlfiles/7407922/change/) (PT)

as can be confirmed by inspecting the stored HTML.

![Screenshot 2024-03-12 at 19 06 32](https://github.com/freelawproject/courtlistener/assets/486004/aaa94540-2852-48eb-9856-478720c9507d)
![Screenshot 2024-03-12 at 19 06 42](https://github.com/freelawproject/courtlistener/assets/486004/999c43d7-82dc-40a3-af89-77484ac5f3b8)

The two dockets were processed 10 minutes apart, so it's unlikely that a `ConflictError` could have caused the update failure.

Upon further investigation, on November 3, we added settings to disable Dockets and RECAPDocuments signals.

I recall we were struggling with celery tasks piling up due to indexing signals that updated everything and not just the fields that changed.

By default, that PR only disabled RECAPDocuments signals.

Reviewing our conversation on Slack, you confirmed on December 8 that signals for Dockets had been disabled for a week. So, I believe that was the reason the docket number was not updated in ES for this Docket.

To confirm this, I processed the two HTML dockets mentioned in this issue, and the docket number was correctly updated in ES.
